static/app/components/profiling/flamegraph/flamegraphUIFrames.tsx

```diff
@@ -213,19 +205,18 @@ export function FlamegraphUIFrames({
       // Only dispatch the zoom action if the new clicked node is not the same as the old selected node.
       // This essentially tracks double click action on a rectangle
       if (lastInteraction === 'click') {
-        if (
-          hoveredNode &&
-          selectedUIFrameRef.current?.length === 1 &&
-          selectedUIFrameRef.current[0] === hoveredNode
-        ) {
-          selectedUIFrameRef.current = [hoveredNode];
-          // If double click is fired on a node, then zoom into it
-          canvasPoolManager.dispatch('set config view', [
-            new Rect(hoveredNode.start, 0, hoveredNode.duration, 1),
-            uiFramesView,
-          ]);
-        }
-
+        // if (
+        //   hoveredNode &&
+        //   selectedUIFrameRef.current?.length === 1 &&
+        //   selectedUIFrameRef.current[0] === hoveredNode
+        // ) {
+        //   selectedUIFrameRef.current = [hoveredNode];
+        //   // If double click is fired on a node, then zoom into it
+        //   canvasPoolManager.dispatch('set config view', [
+        //     new Rect(hoveredNode.start, 0, hoveredNode.duration, 1),
+        //     uiFramesView,
+        //   ]);
+        // }
```

Comment:
what's going on here?

Reply:
I removed this part because it's to track what frame the user clicked on so it zooms into it. Since right now it's possible to click on a piece of canvas where frames overlap, I wasnt sure what to do and I disabled it until i figure it out